source/features/show-names.tsx

```diff
@@ -54,13 +54,12 @@ async function init(): Promise<false | void> {
 			} else {
 				const insertionPoint = usernameElement.parentElement!.tagName === 'STRONG' ? usernameElement.parentElement! : usernameElement;
 				insertionPoint.after(
-					' (',
-					<bdo className="css-truncate d-inline-block">
-						<span className="css-truncate-target" style={{maxWidth: '200px'}}>
+					// That obnoxious part is to comfort the linter
+					<span className="color-text-secondary">{`${'('}`}
+						<bdo className="css-truncate css-truncate-target" style={{maxWidth: '200px'}}>
```

Comment:
The css-truncate class should be on the parent, or else I don't remember if it affects css-truncate-target

Reply:
That CSS rule also works if the two classes are in the same element, however I agree we should keep the original form. An oversight when I was tweaking elements and linter😃